### PR TITLE
Added team data

### LINKED ISSUE #19: Team page
We need a page on which our whole blz-it-team is displayed.

 - [X] Create Page
 - [x] Navigation
 - [x] Translation
 - [x] Proper data format
 - [x] Proper data
 - [ ] Background photo

### PR DESCRIPTION
**Pull request by styrix560**
closes #19

### CODE REVIEW

File: src/pages/[lang]/our-team.astro [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+import { getLangFromUrl, localeParams, useTranslations } from "~/i18n/utils";
+import Hero from "../../components/Hero.astro";
+import TeamPage from "../../components/team/Team.astro";
+import Layout from "../../layouts/Layout.astro";
+import TeamImage from "~/assets/img/team.webp";
+
+export const getStaticPaths = localeParams;
```
Comment (JonasWanke) [inline src/pages/[lang]/our-team.astro:8]: What's this for?

File: src/components/team/PersonImage.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+export type PersonImageProps = {
+  img?: string;
+};
+
+export const PersonImage = ({ img }: PersonImageProps) => {
+  const padding = !img ? "p-5" : "";
```
Comment (JonasWanke) [inline src/components/team/PersonImage.tsx:6]: You can use [clsx](https://www.npmjs.com/package/clsx) (already a dependency)

File: src/assets/data/team.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    "img": "/src/assets/img/team/Benjamin Frost.webp",
+    "name": "Benjamin Frost",
+    "lastRole": "Jury",
+    "hasMultipleRoles": true
+  },
+  {
+    "img": "/src/assets/img/team/Jonas Wanke.webp",
+    "name": "Jonas Wanke",
+    "lastRole": "Expert",
+    "skill": "Skill 08",
```
Comment (JonasWanke) [inline src/assets/data/team.json:12]: Please use the skill names and not numbers

File: src/assets/data/team.json [hunk 4]
```diff
@@ -2,69 +2,69 @@
...
     "name": "Benjamin Frost",
-    "lastRole": "Jury",
+    "lastRole": { "de": "Jury", "en": "Jury" },
     "hasMultipleRoles": true
   },
   {
     "img": "/src/assets/img/team/Jonas Wanke.webp",
     "name": "Jonas Wanke",
-    "lastRole": "Expert",
+    "lastRole": { "en": "Expert", "de": "Experte" },
```
Comment (JonasWanke) [inline src/assets/data/team.json:11]:
```suggestion
    "lastRole": { "en": "Expert", "de": "Bundestrainer" },
```

File: src/assets/data/team.json [hunk 5]
```diff
@@ -2,69 +2,69 @@
...
     "skill": "Skill 08",
     "hasMultipleRoles": true
   },
   {
     "img": "/src/assets/img/team/Dr. Olaf Kappler.webp",
     "name": "Dr. Olaf Kappler",
-    "lastRole": "Expert",
+    "lastRole": { "en": "Expert", "de": "Experte" },
```
Comment (JonasWanke) [inline src/assets/data/team.json:18]:
```suggestion
    "lastRole": { "en": "Expert", "de": "Bundestrainer" },
```

File: src/assets/data/team.json [hunk 6]
```diff
@@ -2,69 +2,69 @@
...
     "skill": "Skill 09"
   },
   {
     "img": "/src/assets/img/team/Doreen Kappler.webp",
     "name": "Doreen Kappler",
-    "lastRole": "Good Fairy"
+    "lastRole": { "de": "Good Fairy", "en": "Good Fairy" }
```
Comment (JonasWanke) [inline src/assets/data/team.json:24]:
```suggestion
    "lastRole": { "de": "Good Fairy", "en": "Gute Fee" }
```

File: src/assets/data/team.json [hunk 7]
```diff
@@ -2,69 +2,69 @@
...
   },
   {
     "img": "/src/assets/img/team/Glenn Skrzypczak.webp",
     "name": "Glenn Skrzypczak",
-    "lastRole": "Expert",
+    "lastRole": { "en": "Expert", "de": "Experte" },
```
Comment (JonasWanke) [inline src/assets/data/team.json:29]:
```suggestion
    "lastRole": { "en": "Expert", "de": "Bundestrainer" },
```

File: src/assets/data/team.json [hunk 8]
```diff
@@ -2,69 +2,69 @@
...
     "name": "Justin Konratt",
-    "lastRole": "Jury",
+    "lastRole": { "de": "Jury", "en": "Jury" },
     "hasMultipleRoles": true
   },
   {
     "img": "/src/assets/img/team/Kai Redmann.webp",
     "name": "Kai Redmann",
-    "lastRole": "Workshop Manager"
+    "lastRole": { "en": "Workshop Manager", "de": "Workshop Manager" }
```
Comment (JonasWanke) [inline src/assets/data/team.json:58]:
```suggestion
    "lastRole": { "en": "Workshop Manager", "de": "Workshop-Manager" }
```

File: src/assets/data/team.json [hunk 9]
```diff
@@ -2,69 +2,69 @@
...
   },
   {
     "img": "/src/assets/img/team/Lukas Fischer.webp",
     "name": "Lukas Fischer",
-    "lastRole": "Workshop Manager"
+    "lastRole": { "en": "Workshop Manager", "de": "Workshop Manager" }
```
Comment (JonasWanke) [inline src/assets/data/team.json:63]:
```suggestion
    "lastRole": { "en": "Workshop Manager", "de": "Workshop-Manager" }
```

File: src/components/team/Person.astro [hunk 10]
```diff
@@ -4,7 +4,7 @@ import { PersonImage } from "./PersonImage";
 
 interface Props {
   name: string;
-  lastRole: string;
+  lastRole: { de: string; en: string };
```
Comment (JonasWanke) [inline src/components/team/Person.astro:7]: This should also work:

```suggestion
  lastRole: Record<Language, string>;
```

**Review by JonasWanke — CHANGES_REQUESTED** [review]

File: src/assets/data/team.json (hunk 4 above)
Comment (styrix560) [reply src/assets/data/team.json:11]: I listened to you when you told me how to translate this, I liked the change and I still did it wrong

File: src/assets/data/team.json (hunk 6 above)
Comment (styrix560) [reply src/assets/data/team.json:24]: "Good Fairy" was the original German (i.e. on our PowerPoints we use it as well). Are we sure about this change? I do not mind either way, I just think the English version sounds better

**Comment by styrix560** [comment]
What do you think about sorting by last names? for example something like:

`name.split(" ").at(-1)`

**Comment by JonasWanke** [comment]
@styrix560 No strong preference about sorting by first vs. last name.

Can you please push your changes? :)

**Comment by styrix560** [comment]
> @styrix560 No strong preference about sorting by first vs. last name.
> 
> Can you please push your changes? :)

Sorry ^^

File: src/assets/data/team.json [hunk 11]
```diff
@@ -8,15 +8,15 @@
   {
     "img": "/src/assets/img/team/Jonas Wanke.webp",
     "name": "Jonas Wanke",
-    "lastRole": { "en": "Expert", "de": "Experte" },
-    "skill": "Skill 08",
+    "lastRole": { "en": "Expert", "de": "Bundestrainer" },
+    "skill": "Mobile Applications Development",
```
Comment (JonasWanke) [inline src/assets/data/team.json:12]: The skill names also need to be localized:

```suggestion
    "skill": {
      "de": "App-Entwicklung",
      "en": "Mobile Applications Development"
    },
```

File: src/components/team/PersonImage.tsx [hunk 12]
```diff
@@ -1,12 +1,16 @@
...
 };
 
 export const PersonImage = ({ img }: PersonImageProps) => {
-  const padding = !img ? "p-5" : "";
   return (
     <img
-      className={`mx-auto h-40 w-40 rounded-full object-cover shadow-lg xl:h-56 xl:w-56 ${padding}`}
+      className={clsx(
+        `mx-auto h-40 w-40 rounded-full object-cover shadow-lg xl:h-56 xl:w-56`,
+        !img && "p-5",
```
Comment (JonasWanke) [inline src/components/team/PersonImage.tsx:12]:
```suggestion
        img || "p-5",
```

**Review by JonasWanke — CHANGES_REQUESTED** [review]

File: src/assets/data/team.json (hunk 11 above)
Comment (styrix560) [reply src/assets/data/team.json:12]: How does this apply to the other skills? Are they fine the way they are or do we need localized names for them as well?

**Review by JonasWanke — APPROVED** [review]

**Comment by styrix560** [comment]
I will also incorperate #302 for team.json

**Pull request merged**